modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinRetrofitCodegen.java

```diff
@@ -0,0 +1,979 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
+ * Copyright 2019 kroegerama
```

Comment:
For new generators (or files), we only keep line 2.

We credit individual template creator in https://github.com/OpenAPITools/openapi-generator#template-creator. Please feel free to add your Github account and new generator there instead.